website/content/docs/overview/what-is-boundary.mdx

```diff
@@ -1,27 +1,22 @@
 ---
 layout: docs
 page_title: What is Boundary?
-description: And introduction to Boundary
+description: An introduction to Boundary
 ---
 
 # What is Boundary?
 
-HashiCorp Boundary is a tool for managing identity-based access for modern, dynamic infrastructure. Boundary’s workflow layers security controls and integrations on multiple levels monitoring and managing user access through:
+HashiCorp Boundary is an agentless identity-aware proxy. Boundary is intended to simplify and secure enabling least-privileged access to cloud infrastructure.
```

Comment:
I think "agentless identity-aware proxy" may be too technical for this page as its many people's first impressions of Boundary